Tests/SwiftProtobufTests/Test_Map.swift

```diff
@@ -68,6 +68,7 @@ class Test_Map: XCTestCase, PBTestHelpers {
         assertDecodeSucceeds([10, 4, 8, 1, 16, 2]) {
             $0.mapInt32Int32 == [1: 2]
         }
+        // TODO: Why should this fail?  It appears to actually be valid.
```

Comment:
Updated the second commit to disable this assert and opened #334 to make sure we have this right.